### ISSUE TITLE
Maintainer wanted

### ISSUE THREAD

**Issue by laktak** [hjson/hjson-py#30]
This project has no maintainers at the moment. If you are interested let us know (@dqsully @trobro).

**Comment by groodt** [comment]
I’m happy to become a maintainer. Let me know how to proceed.

**Comment by laktak** [comment]
@groodt That's great!

Are you interested in maintaining this project for a longer period of time than just a few months? This project is quite stable but please be ready to regularly invest some time into it.

We would expect you to

- keep the project up to date with the Python platform
- work on the new/open issues/PRs and decide what to do about them
- talk to the team about any ideas you have moving forward
- do not make breaking changes
- if breaking changes are necessary you should prepare the best path for users to handle them
- do some basic testing on changes you make/accept
- get to know the codebase

Are you OK with this? Let me know and I'll invite you to the team.

**Comment by groodt** [comment]
@laktak Yes, I'm comfortable with the above. I see the project as being mostly stable but requiring periodic updates to keep modern and respond to minor issues.

**Comment by laktak** [comment]
Great! - I've sent you an invitation to the hjson org. Please accept it (public if possible, private is the default).
